Firebase.agent.lib.nut

```diff
@@ -592,20 +636,32 @@ class Firebase {
     }
 
     // process the http response accordingly
-    function _processResponse (request,callback) {
+    function _invokeCallback(request, callback) {
```

Comment:
The function name is misleading. It would probably be better to rename it to something like `_sendRequest`